### DISCUSSION TITLE
Storage

### DISCUSSION THREAD

**Discussion by NanisTe** [xcv58/Tab-Manager-v2#819] [Q&A]
How does the extension store the relevant data? Is it stored inside the bookmarks?

**Comment by xcv58** [comment]
It uses the browser extensions storage API to persist the data. And the only data it stores is the user preference like whether show URL, Highlight Duplicated Tabs, etc.

API reference:
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/storage
https://developer.chrome.com/docs/extensions/reference/storage/
